server/src/main/java/com/cloud/storage/VolumeApiServiceImpl.java

```diff
@@ -564,7 +567,7 @@ private boolean validateVolume(Account caller, long ownerId, Long zoneId, String
             _resourceLimitMgr.checkResourceLimit(_accountMgr.getAccount(ownerId), ResourceType.secondary_storage);
         }
 
-        sanitizeFormat(format);
+        checkFormatWithSupportedHypervisorsInZone(format, zoneId);
```

Comment:
should we still call sanitizeFormat?

Reply:
yes @DaanHoogland, it's called before. this is duplicate call. so removed